client/src/components/UserTrack/UserTrack.jsx

```diff
@@ -63,6 +64,16 @@ function UserTrack ({
     }} className='icon' size={30}/>
   }
 
+  if (similarityScore >= 80) {
```

Comment:
I'd probably put something like in a helper method. That also makes it easier to test down the line.